configure.ac

```diff
@@ -632,14 +684,14 @@ AM_CONDITIONAL([ENABLE_CAPABILITY], [test "${with_libcap}" = "yes"])
 
 AC_MSG_CHECKING([if cloud functionality should be enabled])
 AC_MSG_RESULT([${enable_cloud}])
-if test "$enable_cloud" != "no"; then
+if test "$enable_cloud" != "no" -a "${enable_wolfssl}" != "yes"; then
     # just to have all messages that can fail ACLK build in one place
     # so it is easier to see why it can't be built
     if test -n "${SSL_LIBS}"; then
         OPTIONAL_SSL_CFLAGS="${SSL_CFLAGS}"
         OPTIONAL_SSL_LIBS="${SSL_LIBS}"
-    else
-        AC_MSG_WARN([OpenSSL required for agent-cloud-link but not found. Try installing 'libssl-dev' or 'openssl-devel'.])
+else
+        AC_MSG_WARN([OpenSSL required for agent-cloud-link but not found. Try installing 'libssl-dev', 'libwolfssl-dev' or 'openssl-devel'.])
```

Comment:
The agent-cloud link doesn't work with `libwolfssl-dev`. You have a check earlier `-a "${enable_wolfssl}" != "yes"`.

Reply:
Good catch, thanks! In reality we only need the openssl libs, but if users give the parameter we should avoid to compile.